Workbooks/PostgreSQL Flexible Server/Performance/High Memory Usage/High Memory Usage.workbook

```diff
@@ -1689,7 +1691,7 @@
                     "query": "let graphgrain=iff({TimeRange:grain}<15m,15m,{TimeRange:grain});\r\nlet TimeStart = todatetime('{GlobalStartTime}');\r\nlet TimeEnd = todatetime('{GlobalEndTime}');\r\nlet TopQueryIDs = \r\n{QueryStoreTable}\r\n| where EndTime >= TimeStart and EndTime <= TimeEnd\r\n| where UserId != 10 // exclude azure super user \r\n|extend Mean_memory_blks=(SharedBlksHit + SharedBlksDirtied)/Calls\r\n| summarize Max_blks = max(Mean_memory_blks) by QueryId, Resource \r\n| where Max_blks>0\r\n| top {TopTimeConsumingQueries} by Max_blks desc\r\n| project QueryId;\r\n{QueryStoreTable}\r\n| where EndTime >= TimeStart and EndTime <= TimeEnd            \r\n| where QueryId in (TopQueryIDs)\r\n|extend Mean_memory_blks=(SharedBlksHit + SharedBlksDirtied)/Calls\r\n| extend  EVENT_TIME_GRAPHGRAIN = bin(todatetime(EndTime), 1m)\r\n| make-series maxmemorytime=max(Mean_memory_blks*8) default=0 on EVENT_TIME_GRAPHGRAIN from TimeStart to TimeEnd step graphgrain by QueryId=strcat('QueryId ',QueryId)",
                     "size": 0,
                     "aggregation": 2,
-                    "title": "Top {TopTimeConsumingQueries} queries by Data Usage. Chart is plotted at a granularity of {ChartGrain}",
+                    "title": "Top {TopTimeConsumingQueries} Queries by Data Usage. Chart is plotted at a granularity of {ChartGrain}",
```

Comment:
in most of the workbooks is being used the syntax "Data is plotted" instead of "Chart is plotted". Can you change here as well for consistency.